src/Application/DirectoryResolver.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application;
+
+use Symfony\Component\Console\Input\InputInterface;
+
+/**
+ * @internal
+ */
+final class DirectoryResolver
+{
+    public static function resolve(InputInterface $input): string
+    {
+        /** @var string $directory */
+        $directory = $input->getArgument('directory') ?? (string) getcwd();
+
+        if ($directory[0] !== DIRECTORY_SEPARATOR && preg_match('~\A[A-Z]:(?![^/\\\\])~i', $directory) === 0) {
```

Comment:
What is this regex for? :)

Reply:
Honestly, I don't know. It was used in AnalyseCommand, I just moved it.

I suppose it check if it's full path or relative path :wink:

Reply:
Ah, sure thing :) Thought you added it, but let's keep it then. If it works, it works ;)